src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.html

```diff
@@ -161,7 +161,7 @@
   </div>
   <div class="steps-actions">
     <div class="steps-action">
-      <button (click)="movePrevious()" data-nav="previous" class="button" id="previous">Back</button>
+      <button (click)="movePrevious()" data-nav="previous" class="button" id="previous">Cancel & Back</button>
```

Comment:
What is Cancel & Back?

The request was to say "Cancel" when on very first step! "Back" when on next steps!

Same way we say Next on each step and Done on last step!